spec/features/sessions/user_can_log_in_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe "a user can log in" do
+  scenario "a user visits login page and logs in" do
+    user = User.create(name: "Bob", email: "cats@cats.cats", password: "cats")
+    visit login_path
+
+    fill_in "email", with: "#{user.email}"
+    fill_in "password", with: "#{user.password}"
```

Comment:
We should update this test after we add the password confirmation field and add a sad path test for if the passwords don't match or one is left blank.